system/include/emscripten/wasmfs.h

```diff
@@ -29,11 +29,13 @@ backend_t wasmfs_get_backend_by_fd(int fd);
 // TODO: Remove this function so that only directories can be mounted.
 int wasmfs_create_file(const char* pathname __attribute__((nonnull)), mode_t mode, backend_t backend);
 
-// Creates a new directory using a specific backend.
-// Returns 0 on success like `mkdir`, or a negative value on error.
-// TODO: Add an alias with wasmfs_mount.
+// Legacy function. This function works like `mkdir` + `wasmfs_mount`.
 int wasmfs_create_directory(const char* path __attribute__((nonnull)), mode_t mode, backend_t backend);
 
+// Mount a backend at a given location in the filesystem
+// `target` must be a valid existing directory.
+int wasmfs_mount(const char* path __attribute__((nonnull)), backend_t backend);
```

Comment:
What is "target" in the docs? path? Or a child under path?

Reply:
Oops, that should be `path`